tools/cldr-apps/js/src/esm/cldrVote.mjs

```diff
@@ -553,18 +558,24 @@ function oneLessPendingVote() {
 function isBusy() {
   if (pendingVoteCount > 0) {
     if (CLDR_VOTE_DEBUG) {
-      console.log("cldrVote busy: pendingVoteCount = " + pendingVoteCount);
+      console.log(
+        "cldrVote busy: pendingVoteCount = " +
+          pendingVoteCount +
+          " time = " +
+          Date.now()
+      );
     }
     return true;
   }
-  if (Date.now() - lastTimeChanged < 3000) {
+  const time = Date.now();
+  if (time - lastTimeChanged < 3000) {
     if (CLDR_VOTE_DEBUG) {
-      console.log("cldrVote busy: less than 3 seconds elapsed");
+      console.log("cldrVote busy: less than 3 seconds elapsed; time = " + time);
     }
     return true;
   }
   if (CLDR_VOTE_DEBUG) {
-    console.log("cldrVote not busy");
+    console.log("cldrVote not busy; time = " + time);
```

Comment:
a logging framework  ( <https://getpino.io/#/> is one i've used / contributed to  though not in browser , or something homegrown) could collect timings automatically